docs/conf.py

```diff
@@ -12,17 +12,46 @@
 
 autodoc_mock_imports = ["f90nml", "jsonschema", "lxml"]
 copyright = str(dt.datetime.now().year)
-extensions = ["sphinx.ext.autodoc", "sphinx.ext.intersphinx"]
+extensions = ["sphinx.ext.autodoc", "sphinx.ext.extlinks", "sphinx.ext.intersphinx"]
+extlinks_detect_hardcoded_links = True
 html_logo = os.path.join("static", "ufs.png")
 html_static_path = ["static"]
 html_theme = "sphinx_rtd_theme"
 intersphinx_mapping = {"python": ("https://docs.python.org/3", None)}
+linkcheck_anchors_ignore = ["cloning-a-fork"] # silence warnings about valid anchors
```

Comment:
This is unfortunate. The link checker flags some URLs with `#anchors` as invalid, even though the URLs actually load fine in the browser. There's `wontfix` [issue about it](https://github.com/sphinx-doc/sphinx/issues/9255) -- apparently it may be the case that the remote page content is generated by JavaScript such that the anchor in question isn't seen in a static load of the page. Currently, there's one GitHub page+anchor that exhibits this problem. I don't like this, but don't know what else to do about it. This works.